linodecli/overrides.py

```diff
@@ -37,3 +41,60 @@ def handle_domains_zone_file(operation, output_handler, json_data) -> bool:
     """
     print("\n".join(json_data["zone_file"]))
     return False
+
+
+@output_override("linodes", "types", OutputMode.table)
+def handle_types_region_prices_list(
+    operation, output_handler, json_data
+) -> bool:
+    # pylint: disable=unused-argument
+    """
+    Override the output of 'linode-cli linodes types' to display regional pricing.
+    """
+    if len(json_data["data"]) < 1:
+        return True
+
+    output = Table()
+
+    # To ensure the order of the headers and make sure we have region_prices as the last column
+    headers = sorted(json_data["data"][0].keys() - ["addons", "price", "region_prices"], key=len)
+    headers += ["price.hourly", "price.monthly", "region_prices"]
+    region_price_sub_headers = ["id", "hourly", "monthly"]
+
+    for header in headers:
+        output.add_column(header, justify="center")
+
+    for linode in json_data["data"]:
+        row = []
+        for h in headers:
+            if h == "region_prices":
+                sub_table = Table()
+                for header in region_price_sub_headers:
+                    sub_table.add_column(header, justify="center")
+                for region_price in linode[h]:
+                    region_price_row = []
+                    for header in region_price_sub_headers:
+                        region_price_row += (
+                            Align(str(region_price[header]), align="left"),
+                        )
+                    sub_table.add_row(*region_price_row)
+                row += [sub_table]
+
+            elif h == "price.hourly" or h == "price.monthly":
```

Comment:
Just a small suggestion to make the linter happy, otherwise LGTM!
```suggestion
            elif h in ("price.hourly", "price.monthly"):
```